app/controllers/users_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  include ControllerHelper
+  before_action :user, only: %i[show edit update destroy]
+
+  def index
+    @users = User.page params[:page]
+  end
+
+  def show; end
+
+  def edit; end
```

Comment:
unnecessary methods definition as you already have `before_action`
remove

Reply:
done